### ISSUE TITLE
Settings link not visible in admin sidebar

### ISSUE THREAD

**Issue by FreekVR** [engram-design/ImageResizer#57]
### Description
The admin menu item / tab is not visible.

However, I can view the settings, change them, and use the logs / batch actions if I manually navigate to '/admin/image-resizer/settings'

This may have something to do with having `allowAdminChanges: false;` though I'd expect if that was the cause that navigating to the settings page also wouldn't work.

### Additional info

- Plugin version: 2.0.9
- Craft version: 3.4.27

**Comment by engram-design** [comment]
Not seeing this on my end, but could you update to [2.0.10](https://github.com/verbb/image-resizer/releases/tag/2.0.10).

Just to clarify, we're talking about the icon in the settings page? https://d.pr/i/LyFFiV

There is otherwise no menu item on the admin sidebar for Image Resizer.

**Comment by FreekVR** [comment]
With 'allowAdminChanges' disabled, the settings page is not reachable in Craft.

However, I still can and should be able to reach the pages for batch actions etc. Right now the only way to get there is by manually entering the URL in this scenario.

**Comment by engram-design** [comment]
Right, I'll see what I can do about that, I'll very likely add it to Utilities.

**Comment by darylknight** [comment]
Any news on this? I just installed it to batch resize a bunch of images, deployed to staging and realised I could only use it on my local machine. Perhaps another workaround, or addition, would be to add some kind of console command to run batch resize from the command line?

**Comment by FreekVR** [comment]
@darylknight you should still be able to get there by entering `/admin/image-resizer/settings` in your address bar on your staging site.

**Comment by darylknight** [comment]
Oh that's excellent, thanks!
